src/test/java/core/basesyntax/model/OperationTest.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class OperationTest {
+
+    @Test
+    public void getOperationByFirstLetter_ValidOperationLetter_isOk() {
```

Comment:
I think you should check all operations letters and do some negative tests like with wrong letter, null and number